docs/source-app/core_api/lightning_app/dynamic_work_content.rst

```diff
@@ -103,16 +103,81 @@ In order to do that, we are iterating over the list of ``jupyter_config_requests
 
 .. code-block:: python
 
+    import os
+    import subprocess
     import lightning as L
+    from lightning_app.structures import Dict
+    from lightning_app.storage import Path
+    import sys
+    from typing import Optional
+
+    class JupyterLabWork(L.LightningWork):
+	def __init__(self, cloud_compute: Optional[L.CloudCompute] = None):
+		super().__init__(cloud_compute=cloud_compute, parallel=True)
```

Comment:
Hey @oojo12 Could you fix the indentation in this code sample? It is twice of what it should be.